spec/models/ability_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+require 'cancan/matchers'
+
+RSpec.describe Ability, type: :model do
```

Comment:
not enough tests
move tests from models here

Reply:
done, but the comment did not disappear